source/administration/downgrade.rst

```diff
@@ -0,0 +1,83 @@
+Downgrading Mattermost Server
+=============================
+
+In most cases you can downgrade Mattermost Server using the same steps as :doc:`upgrade`. The binaries can be found in the :doc:`version-archive`. We do not recommend downgrading more than one version back from your current installation.
+
+Downgrade from 5.0 to 4.10
+---------------------------
+
+During the version 5.0 upgrade process, a migration is run to enable new features of the `advanced permissions system <https://docs.mattermost.com/deployment/advanced-permissions.html>`_. This changes the database in ways that result in it no longer being compatible with Mattermost server 4.10. If you need to downgrade from 5.0 or later to 4.10, it is necessary to follow the steps below.
+
+You should gather the following information before starting the upgrade:
+
+Existing install directory - *{install-path}*
+  If you don't know where Mattermost Server is installed, use the ``whereis mattermost`` command. The output should be similar to */opt/mattermost/bin/mattermost*. The install directory is everything before the last occurrence of the string */mattermost*. In this example, the *{install-path}* is ``/opt``.
+Location of your local storage directory
+  The local storage directory contains all the files that users have attached to their messages. If you don't know its location, open the System Console and go to **Files > Storage** and read the value in **Local Storage Directory**. Relative paths are relative to the ``mattermost`` directory. For example, if the local storage directory is ``./data/`` then the absolute path is ``{install-path}/mattermost/data``.
+Owner and group of the install directory - *{owner}* and *{group}*
+  Use the ``ls -l {install-path}/mattermost/bin/mattermost`` command to get the owner and group.
+
+#. In a terminal window on the server that hosts Mattermost Server, change to your home directory. If any, delete files and directories that might still exist from a previous download.
+
+   .. code-block:: sh
+
+     cd ~
+
+#. Stop Mattermost Server.
+
+   On Ubuntu 14.04 and RHEL 6.6:
+
+   .. code-block:: sh
+
+     sudo service mattermost stop
+
+   On Ubuntu 16.04 and RHEL 7.1:
+
+   .. code-block:: sh
+
+     sudo systemctl stop mattermost
+     
+
+     
+#. Back up your data and application.
+
+   #. Back up your database using your organization’s standard procedures for backing up MySQL or PostgreSQL.
+
+   #. Back up your application by copying into an archive folder (e.g. ``mattermost-back-YYYY-MM-DD-HH-mm``).
+
+    cd {install-path}/mattermost
+    sudo setcap cap_net_bind_service=+ep ./bin/mattermost
```

Comment:
Is this step really need in this stage? also the formatting is a bit strange, it is using a quote